train.py

```diff
@@ -83,7 +83,7 @@ def eval(model, criterion, data, fields):
     return stats
 
 
-def train_model(model, train_data, valid_data, fields, optim):
+def train_model(model, train_data, valid_data, fields):
```

Comment:
Can we also move this to a onmt/Trainer.py file with anything else we might need for training and optimization. Would like to be able to train without using the command-line.